### PR TITLE
made changes in config

### PR DESCRIPTION
**Pull request by rohitmishra209**

### CODE REVIEW

File: lib/util/configMerge.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * MIT Licensed
+ */
+var config = require('../../config');
+var util = require('./index');
+
+
+config = util.buildAppConfig(config);
+util.validateConfig(config);
+
+exports.getConfig = function () {
```
Comment (ninadhatkar) [inline lib/util/configMerge.js]:
```suggestion
module.exports = function () {
```

File: lib/util/login.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ var request = require('./request');
 var pkg = require('../../package');
 
 module.exports = function (config) {
+  console.log("condiffff", config);
```
Comment (ninadhatkar) [inline lib/util/login.js]: Remove console

File: lib/util/index.js [hunk 3]
```diff
@@ -82,3 +85,15 @@ exports.sanitizeStack = function (config) {
       }
     });
 };
+
+exports.getConfig = function() {
+  config = this.buildAppConfig(config);
+  // eslint-disable-next-line no-undef
```
Comment (ninadhatkar) [inline lib/util/index.js]: Can we remove this file?

File: lib/util/index.js [hunk 4]
```diff
@@ -82,3 +85,15 @@ exports.sanitizeStack = function (config) {
...
+  // buildAppConfig(config);
```
Comment (ninadhatkar) [inline lib/util/index.js]: Can we remove this line?

File: lib/util/configMerge.js (hunk 1 above)
Comment (rohitmishra209) [reply lib/util/configMerge.js]: done

File: lib/util/login.js (hunk 2 above)
Comment (rohitmishra209) [reply lib/util/login.js]: Done

File: lib/util/index.js (hunk 3 above)
Comment (rohitmishra209) [reply lib/util/index.js]: Done

**Pull request merged**